test/common/reference_data_wrangler.hpp

```diff
@@ -276,6 +276,18 @@ void verify_dft(const portfft::descriptor<Scalar, Domain>& desc, std::vector<Ele
       throw std::runtime_error("Verification Failed");
     }
   }
+#ifdef PORTFFT_LOG_DUMPS
+  std::cout << "ref_output: ";
+  for (std::size_t t = 0; t < ref_output.size(); ++t) {
```

Comment:
It might be of value to create a file and print all the values there rather than just dumping everything on the terminal. That's how I was debugging while implementing the global implementation and was much much simpler. 

For example, here we can create a file called, let's say reference_output.txt and print everything there.
Similarly, where we are planning to print data after each factor in global implementation, we can name those files as factor_1.txt and so on...

Reply:
If the output is too big for terminal you can always pipe it to a file.